g2core/spindle.cpp

```diff
@@ -369,6 +375,23 @@ void spindle_end_override(const float ramp_time)
     return;
 }
 
+// Called in ISR so be careful and fast
+void spindle_update_laser_override(float current_velocity) {
+    float spindle_override = 0.0;
+    float feed_rate = cm_get_feed_rate(ACTIVE_MODEL);
+
+    if (feed_rate > 0.0) {
+        spindle_override = current_velocity / feed_rate;
+        if (spindle_override > 1.0) spindle_override = 1.0;
+        else if (spindle_override < 0.0) spindle_override = 0.0; 
+    }
+//    if (fabs(spindle_override - spindle.override_factor) >= 0.01) {
```

Comment:
Should this commented out line be nuked?

Reply:
I will remove these lines.  I was trying to reduce the amount of updating done to the PWM but it seems it isn't an issue.